### ISSUE TITLE
removing google sign in modal from chrome://devices

### ISSUE THREAD

**Issue by kjozwiak** [brave/brave-browser#1279]
## Description 

We should remove the Google `Sign In` modal window under `chrome://devices` which also includes Google branding. Because this is pretty buried, I don't think we need to fix it for `0.55.x` but should remove this before releasing `1.0.`.

## Steps to Reproduce

   1. launch `b-c` and load `chrome://devices`
   2. click on `sign in` and you'll get a Google sign in modal that includes Google branding

## Actual result:

<img width="905" alt="screen shot 2018-09-25 at 4 07 32 pm" src="https://user-images.githubusercontent.com/2602313/46040728-240ef480-c0df-11e8-8ba9-6e88c5344f25.png">

## Expected result:

Clicking on `Sign in` under `chrome://devices` shouldn't launch the Google modal which also includes Google branding.

## Reproduces how often: 

100% reproducible when going through the above STR.

## Brave version (chrome://version info)

Brave | 0.55.6 Chromium: 70.0.3538.16 (Official Build) dev (64-bit)
-- | --
Revision | 16ed95b41bb05e565b11fb66ac33c660b721f778-refs/branch-heads/3538@{#306}
OS | Mac OS X

### Reproducible on current release:

Yes, currently reproducible with `0.55.6 Chromium: 70.0.3538.16` which is the current release.

**Comment by hferreiro** [comment]
https://github.com/brave/brave-core/commit/a0609ecc79288559ae4fbe16dee592f403b88458 just removes the 'sign in' link but does it make sense to show the rest of the sentence?

**Comment by bbondy** [comment]
We should probably remove the full line of text, thanks for catching it.

**Comment by bbondy** [comment]
Please do a separate PR for that, you can link it to the same issue.

**Comment by hferreiro** [comment]
The 'Add printers' button also redirects to a Google sign in page. I guess the 'Classic printers' dev should be deleted entirely, right @bbondy?

**Comment by srirambv** [comment]
After #1312 got into Beta builds this is what i see when i click on `Sign in` button so its better to remove that entire section itself. Am sure other sign in links(https://github.com/brave/brave-browser/issues/1363) would show up the same 
![image](https://user-images.githubusercontent.com/17010094/46322234-be76a880-c5b5-11e8-84db-b33d2d7762d2.png)

**Comment by kjozwiak** [comment]
> The 'Add printers' button also redirects to a Google sign in page. I guess the 'Classic printers' div should be deleted entirely, right @bbondy?

I would agree and say we remove the entire `classic printers` div but I'll let @bbondy make the call 👍

**Comment by srirambv** [comment]
Once you get this popup with `no-thanks.invalid` , clicking outside doesn't close it, have to use Esc button to close it. Reload doesn't remove as well

**Comment by kjozwiak** [comment]
Went through verification using the following build under `macOS 10.13.6 x64` - **PASSED**

Brave | 0.55.12 Chromium: 70.0.3538.45 (Official Build) beta(64-bit)
-- | --
Revision | cbdc32e4334458954e9def214d7e5fa1ca1960eb-refs/branch-heads/3538@{#830}
OS | Mac OS X

<img width="796" alt="screen shot 2018-10-10 at 9 52 27 am" src="https://user-images.githubusercontent.com/2602313/46741079-4291f580-cc72-11e8-9dc9-9ef58b9febcb.png">
